src/safe_cli/operators/safe_tx_service_operator.py

```diff
@@ -45,7 +44,7 @@ def approve_hash(self, hash_to_approve: HexBytes, sender: str) -> bool:
 
     def sign_message(
         self,
-        eip191_message: Optional[str] = None,
+        eip191_message: Optional[bool] = False,
```

Comment:
I can't find the use of the parameter eip191_message, is it correct here?

Reply:
Good catch! It is a mutual exclude option, I think that we can remove it